NEWS.md

```diff
@@ -88,6 +88,7 @@
 ## Notes
 
 * All user-facing messages are now prepared using the `{cli}` package (#2418, @IndrajeetPatil). All messages have been reviewed and updated to be more informative and consistent.
+* Error messages and linters contain clickable hyperlinks to improve code navigation (#2645, #2588, @olivroy).
```

Comment:
```suggestion
* File locations in error messages and lints contain clickable hyperlinks to improve code navigation (#2645, #2588, @olivroy).
```

Reply:
Do we expect this to work in any IDE? I guess in terminal it tries to open the file with `EDITOR` (sorry can't test right now)